content/en/continuous_integration/troubleshooting.md

```diff
@@ -142,6 +142,8 @@ The default branch is used to power some features of the products, namely:
 
 - New flaky tests: Tests that are not currently classified as flaky in the default branch. If the default branch is not properly set, this could lead to a wrong number of detected new flaky tests.
 
+- Pipelines list: only default branches are displayed there, so a branch that is not results in a missing item from this table or the wrong branch being displayed.
```

Comment:
```suggestion
- Pipelines list: The pipelines list only displays default branches. Setting the wrong default branch can result in missing or incorrect data in the pipelines list.
```
I realize you copied this description from the "Default branches list" item above, but that one could also use improvement. If you'd like to improve the docs, you can update that item to the following:
- Default branches list: This list only displays default branches. Setting the wrong default branch can result in missing or incorrect data in the default branches list.

Reply:
Thanks for the review! Of course I want to improve the docs, applied your suggestion to both.